gnxi_tester/README.md

```diff
@@ -2,15 +2,35 @@
 
 A CLI tool for orchestrating tests against a gNXI target.
 
-## Install
+## CLI
+
+### Installing
 
 ```
 go get github.com/google/gnxi/gnxi_tester
 go install github.com/google/gnxi/gnxi_tester
 ```
 
-## Run
+### Running
+
+Tests are provided in `~/.gnxi.yml`. By default, the following processes & clients will be run: 

```

Comment:
any pointer for the format of .gnxi.yml ?
will it be automatically generated? Or is there an example?

Reply:
It will be auto generated by viper. So the defaults in config/defaults.go will be written to it.